packages/devtools-network-request/networkRequest.js

```diff
@@ -1,13 +1,10 @@
-const { getValue } = require("devtools-config");
-
 // opts is ignored because this is only used in local development and
 // replaces a more powerful network request from Firefox that can be
 // configured.
 function networkRequest(url, opts) {
-  const devServerURL = getValue("host");
-
+  const devServerOrigin = location.origin;
```

Comment:
At this point why not just `fetch("/get....")`? which is how it was before the app stuff.

So the app is currently just showing the webpage from the server, yeah, which is why I didn't think this was needed. And this will work when it's locally running the server itself too.

Makes sense, thanks for thinking through it!